content/new-leads/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Kickstart your own community
+---
+
+There are many communities stablished within Galaxy but if you feel like you  just don't see your own or don't feel identified with them you can create your own community!
+
+Below you'll find a suggested path to follow before you can start your own community, a list of Do's and Dont's and also useful links you should probably check out.
+
+### Path 
+
+#### Learn the basics about Galaxy
+
+First of all, you should  start with our dedicated [Get Started](/get-started/) page to get a head-start of what Galaxy is about. You'll find all the necessary resources and you could even participate in the [Galaxy Mentorship Network](#) to get started with a Galaxy Mentor.
+
+#### Learn about Tool Development
+Galaxy as many tools already installed and ready to be used but when creating your community you'll probably need specific tools for your Galaxy analysis. Don't worry, see our dedicated page for [Tool Authors](/tools/) where you'll find everything you'll need. Don't forget to follow the [Best Practices](https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices.html) for writing Galaxy Tools.
```

Comment:
Here I would suggest they reach out to the "Tools" working group. To develop your own tool, you often need to set-up your own tool repository in github, account to push to the Galaxy Toolshed, etc.. These are not necessary obvious and getting some guidance from the "tools" working group could be useful (but I don't know if it is their role). Someone (Bjoern) set up the repo, etc. for me.